packages/lesswrong/components/forumEvents/ForumEventFrontpageBanner.tsx

```diff
@@ -69,10 +157,41 @@ const styles = (theme: ThemeType) => ({
   description: {
     ...forumEventBannerDescriptionStyles(theme),
   },
+  titleWithPoll: {
+    fontSize: 32,
+    fontWeight: 700,
+    lineHeight: 'normal',
+  },
+  titleWithPollMobile: {
+    fontSize: 28,
+    fontWeight: 700,
+    lineHeight: 'normal',
+  },
+  dateWithPoll: {
+    fontSize: 14,
+    fontWeight: 600,
+    lineHeight: 'normal',
+    marginTop: 6,
+  },
+  descriptionWithPoll: {
+    fontSize: 13,
+    fontWeight: 500,
+    lineHeight: '140%',
+    marginTop: 16,
+    "& a": {
+      textDecoration: "underline",
+      textUnderlineOffset: '2px',
+    }
+  },
   image: {
     position: "absolute",
     zIndex: -1,
-    top: "-45%",
+    top: -HEADER_HEIGHT,
+    right: 0,
+    width: '100%',
+  },
+  imageWithGradient: {
+    // top: "-45%",
```

Comment:
Oops I wanted to test this, will do so tomorrow

Reply:
Ok looking at a standard forum event with and without this, I think Agnes just added it to adjust a specific banner, so I'm going to remove it since it's not a general style we want